code/parachain/runtime/common/src/lib.rs

```diff
@@ -226,11 +224,8 @@ pub fn multi_existential_deposits<AssetsRegistry: AssetRatioInspect<AssetId = Cu
 		NativeExistentialDeposit::get(),
 	)
 	// TODO:
-	// 1. ask approved DEX pair for price  (is it enough performance? or should we allow pay in
-	// ED PICA for other asset account?)
-	// 2. ask CurrencyFactory
-	// 3. use hardcoded values
-	// 4. else 1_000_000_u128
+	// 1. ask CurrencyFactory
+	// 2. use hardcoded values
```

Comment:
Should hard-coded checks come first before we try and query currency factory?

Reply:
same for AR. if we allow to override. if we allow to override then CF should be first.